pos/modules/checkout/hooks/useKhanCard.tsx

```diff
@@ -53,7 +53,7 @@ export const useSendTransaction = ({
   onCompleted,
   onError,
 }: {
-  onCompleted: () => void
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Reevaluate the use of `any` for the `onCompleted` parameter type.

```suggestion
onCompleted: (info?: unknown) => void
```